enterprise-suite/es-grafana/kubernetes-exporter.json

```diff
@@ -1,7 +1,32 @@
 [
-   {"graphName": "CPU Percent", "promQL":["node_cpu_percent"]},
-   {"graphName": "Memory Available Percent", "promQL":["node_memory_usable_percent"]},
-   {"graphName": "File System Free Percent", "promQL":["node_filesystem_free_percent"]},
-   {"graphName": "Bytes Received Per Second", "promQL":["sum by (instance) (irate(node_network_receive_bytes[1m]))"]},
-   {"graphName": "Bytes Transmitted Per Second", "promQL":["sum by (instance) (irate(node_network_transmit_bytes[1m]))"]}
+    {
+        "graphName": "CPU Percent",
+        "promQL": [
+            "sum without(container_name,id,image,name)((irate(container_cpu_user_seconds_total{ContextTags}[1m]) + irate(container_cpu_system_seconds_total[1m])) * 100)"
+        ]
+    },
+    {
+        "graphName": "Memory Available Percent",
+        "promQL": [
+            "100 - sum without(container_name,id,image,name)((container_memory_usage_bytes{ContextTags} / container_spec_memory_limit_bytes < +Inf or container_memory_usage_bytes{ContextTags} / on (node_ip) group_left node_memory_MemTotal) * 100)"
+        ]
+    },
+    {
+        "graphName": "File System Free Percent",
+        "promQL": [
+            "100 - sum without(container_name, id, image, name)(container_fs_usage_bytes{ContextTags} / container_fs_limit_bytes * 100)"
+        ]
+    },
+    {
+        "graphName": "Bytes Received Per Second",
+        "promQL": [
+            "sum without(container_name, id, image, name, interface)(irate(container_network_receive_bytes_total{ContextTags}[1m]))"
+        ]
+    },
+    {
+        "graphName": "Bytes Transmitted Per Second",
+        "promQL": [
+            "sum without (container_name,id, image, name, interface)(irate(container_network_transmit_bytes_total{ContextTags}[1m]))"
+        ]
+    }
 ]
```

Comment:
Will this work with the top-level (cluster view) grafana dashboard?

My idea is we should have two separate dashboards - one is the cluster dashboard, that the console home page links to, and the workspace scoped container dashboard (this one here).

Reply:
Ah right, that is in a separate exporter. Nevermind.